src/ralph/backends/data/ldp.py

```diff
@@ -0,0 +1,244 @@
+        self.client = ovh.Client(
+            endpoint=settings.ENDPOINT,
+            application_key=settings.APPLICATION_KEY,
+            application_secret=settings.APPLICATION_SECRET,
+            consumer_key=settings.CONSUMER_KEY,
+        )
+        self.service_name = settings.SERVICE_NAME
+        self.stream_id = settings.DEFAULT_STREAM_ID
+        self.timeout = settings.REQUEST_TIMEOUT
+
+    def status(self) -> DataBackendStatus:
+        """Checks whether the default service_name is accessible."""
+        try:
+            self.client.get(self.get_archive_endpoint())
+        except ovh.exceptions.APIError as error:
+            logger.error("Failed to connect to the LDP: %s", error)
+            return DataBackendStatus.ERROR
+        except BackendParameterException:
+            return DataBackendStatus.ERROR
+
+        return DataBackendStatus.OK
+
+    def list(
+        self, target: str = None, details: bool = False, new: bool = False
+    ) -> Iterator[Union[str, dict]]:
+        """Lists archives for a given target stream_id.
+
+        Args:
+            target (str or None): The target stream_id where to list the archives.
+                If target is `None`, the `DEFAULT_STREAM_ID` is used instead.
+            details (bool): Get detailed archive information instead of just ID's.
+            new (bool): Given the history, list only not already read archives.
+
+        Yields:
+            str: If `details` is False.
+            dict: If `details` is True.
+
+        Raises:
+            BackendParameterException: If the `target` is `None` and no
+                `DEFAULT_STREAM_ID` is given.
+            BackendException: If a failure during retrieval of archives list occurs.
+        """
+        list_archives_endpoint = self.get_archive_endpoint(stream_id=target)
+        logger.debug("List archives endpoint: %s", list_archives_endpoint)
+        logger.debug("List archives details: %s", str(details))
+
+        try:
+            archives = self.client.get(list_archives_endpoint)
+        except ovh.exceptions.APIError as error:
+            msg = "Failed to get archives list: %s"
+            logger.error(msg, error)
+            raise BackendException(msg % error) from error
+
+        logger.debug("Found %d archives", len(archives))
+
+        if new:
+            archives = set(archives) - set(self.get_command_history(self.name, "read"))
+            logger.debug("New archives: %d", len(archives))
+
+        if not details:
+            for archive in archives:
+                yield archive
+
+            return
+
+        for archive in archives:
+            yield self._details(target, archive)
+
+    @enforce_query_checks
+    def read(
+        self,
+        *,
+        query: Union[str, BaseQuery] = None,
+        target: str = None,
+        chunk_size: Union[None, int] = 4096,
+        raw_output: bool = True,
+        ignore_errors: bool = True,
```

Comment:
I agree for `raw_output = True`, as we always returns bytes, but I would have think that we don't ignore errors, so `ignore_errors = False`. WDYT?

Reply:
Ok, let's keep `raw_output = True`

I like the reasoning that if a boolean argument is not used (has any effect), then its default value should be `False`.
On the other hand, setting `raw_output = True` would be helpful to indicate the only supported behavior.
Then, regarding `ignore_errors`, IMO, the behavior would be closer to `True` than `False`, although both work:
We don't ignore errors, which implies that we handle them, but since we don't have any errors to handle, we don't handle them and thus ignore them.